Changes

```diff
@@ -69,6 +69,10 @@ Improvements
 - MeshTangents : Added support for additional computation modes (#3030).
 - GafferImageUI: Added DWA compression presets (#3153).
 - Numeric Bookmarks : Added serialisation to preserve numeric bookmarks across sessions (#3157).
+- SetExpressions : Support wildcard characters in set names (#3172).
+- LightVisualisers : Added "visualiserScale" attribute (#3180).
```

Comment:
Replace with the user-facing part of this : the new `visualiserScale` plug on Light nodes.